packages/shared/kebab/kebab.tsx

```diff
@@ -105,6 +111,13 @@ export const Kebab: React.FC<KebabProps> = ({
 
   const history = useHistory();
 
+  const isAllowed = useAccessReview({
```

Comment:
```suggestion
  const [isAllowed, loading] = useAccessReview({
```
While loading maybe you should disable all items to stop users from accidentally being able to reach the edit page?